LICENSE

```diff
@@ -199,287 +199,3 @@ Apache License
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
```

Comment:
If you are going to remove the thirdparty license into independent files, you should add related pointer here.